salad/features/steps-browser-remember.feature

```diff
@@ -0,0 +1,216 @@
+Feature: Ensuring that the remember / recall steps work
+    In order to make sure that the remember part of the form module works
+    As a developer
+    I test against the form test files
+
+    Scenario: 1. Filling in a field with random content and recalling the partial content
+        Given I visit the salad test url "browser/form.html"
+          And I look around
+         # --- string with length
+         # --- test element contains string / element is string
+         When I store a random <what> of length 20 as "string_with_length"
```

Comment:
Interesting new feature.  Not totally sure what I think about it, but really interesting to see salad developing in this way!